src/bindings/scala/build.sbt

```diff
@@ -19,8 +19,8 @@ import BuildSupport._
 lazy val commonSettings = {
   Seq(
     organization := "com.ctc",
-    scalaVersion := "2.12.13",
-    crossScalaVersions := Seq("2.12.13", "2.13.8"),
+    scalaVersion := "2.13.6",
```

Comment:
Why are you doing this?

Reply:
oh that was because I didn't know about the `sbt +install` for the cross compatibility I meant to revert this back will do that now.